tests/suit/framework/framework.go

```diff
@@ -202,7 +202,7 @@ func (f *Framework) CreateNamespace(prefix string, labels map[string]string) (*v
 
 // AddNamespaceToDelete provides a wrapper around the go append function
 func (f *Framework) AddNamespaceToDelete(ns *v1.Namespace) {
-	f.namespacesToDelete = append(f.namespacesToDelete, ns)
+	//f.namespacesToDelete = append(f.namespacesToDelete, ns)
```

Comment:
why? let's not call it and leave the function as is - or delete it as a whole if we are not going to use that.

Reply:
@liranr23 that's part of the commit that is used for debugging.. there's another task on @eslutsky 's backlog to change this